mkdocs.yml

```diff
@@ -13,16 +13,11 @@ nav:
   - Utilities:
       - utilities/idempotency.md
       - utilities/parameters.md
-      - utilities/large_messages.md
+      - utilities/sqs_large_message_handling.md
```

Comment:
These changes need to be rolled back -  from L16-L25. We released new modules and changed the doc structure after you forked!